front/src/layouts/menu/menu.layout.css

```diff
@@ -30,6 +30,25 @@ ul.main-color > li.main-color.is-active {
   color: var(--auxiliary-bg-color);
 }
 
+ul.main-color > li.main-color:hover {
+  background-color: var(--main-bg-color);
+  color: var(--main-txt-color); 
+  cursor: default;
+}
+
 .el-menu-item > a { 
+  color: var(--main-txt-color);
+  font-size: 16px;
+  padding: 10px 25px;
+  border-radius: 3px;
   text-decoration: none;
-} 
+}
+
+a#menu1:hover,
+a#menu2:hover,
+a#menu3:hover,
+a#menu4:hover,
+a#menu5:hover {
```

Comment:
why do you need ids? 
1. you can use common selector el-menu-item 
2. your rule is common for all items
3. ids are usually for js manipulation, not for styles
4. order-number is not good practice in naming of classes or ids. if you need to specify something for element which connected with order you can use https://www.w3schools.com/cssref/sel_last-of-type.asp or others